lib/ricohapi/mstorage/client.rb

```diff
@@ -120,6 +125,10 @@ def remove_meta(media_id, key)
       private
 
       def handle_response(as_raw = false, retrying = false)
+        if self.auth_client
+          self.token = self.auth_client.api_token_for! RicohAPI::MStorage::SCOPE
+        end
```

Comment:
I think this if-statement can be inline style.
ex) https://github.com/hishida/media-storage-rb/blob/cddfc0e84c0c0a83557e202b13cdbee93177aef4/lib/ricohapi/mstorage/client.rb#L139